### ISSUE TITLE
TSC both screens non-functioning on TBM 930 

### ISSUE THREAD

**Issue by cmr771** [Working-Title-MSFS-Mods/fspackages#1053]
Hello,
I have installed the workingtitle-g3000 into the Community folder and now both TSC screen appear fine, but do not function as touch screens. The three buttons on either side work, but touchscreens themselves are do not react when touched.

As of now, I only have these 3 folders in Community Folder:
**workingtitle-g3000
fspackages-main
aircraft-tbm930-improvement**

I have removed the other two folder from the Community folder to try to isolate the issue, but when the workingtitle-g3000 folder is in there, it disables the TSC from functioning.

Any thoughts would be appreciated.

Thank you

**Comment by cywang90** [comment]
You probably shouldn't have fspackages-main in the community folder. I would remove that and see what happens.

**Comment by cmr771** [comment]
> You probably shouldn't have fspackages-main in the community folder. I would remove that and see what happens.

Thanks for the idea cywang90, however, I had tried removing fspackages-main from the Community folder previously, to no avail. I did try it again now, just in case, but same result. The curser icon changes from the pointer to the small hand icon when I hover over the TSC screens as though it will interact with the touchscreen, but none of the buttons react, however the three push button to the side (PFD, MFD and NAV COM) all work fine.

Thanks again for the suggestion.
